packages/languages/bin/download.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable */
+
+/**
+ * External dependencies
+ */
+const https = require( 'https' );
+const fs = require( 'fs' );
+const path = require( 'path' );
+
+const LANGUAGES_META_URL = 'https://widgets.wp.com/languages/calypso/languages-meta.json';
+const FILE_PATH = path.resolve( __dirname, '..', 'src', 'languages-meta.json' );
+
+https
+	.get( LANGUAGES_META_URL, ( response ) => {
+		response.setEncoding( 'utf8' );
+		response.pipe( fs.createWriteStream( FILE_PATH ) );
+	} )
+	.on( 'error', ( error ) => {
+		if ( process.env.CALYPSO_ENV === 'production' ) {
```

Comment:
Now that the download script is being manually run, it might be better to always print the error and exit the script.